docs/notebooks/dev_guide/grid.ipynb

```diff
@@ -0,0 +1,1225 @@
+{
```

Comment:
<p>θ</p><p>&lt;sup&gt;3 coordinate &gt; π</p><p> Typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaControl/DESC/pull/409/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Yes. That was supposed to be a footnote marker at the end of the sentence.